api/inc/page_allocator_exports.h

```diff
@@ -30,11 +30,6 @@
 #define UVISOR_ERROR_PAGE_INVALID_PAGE_OWNER    (UVISOR_ERROR_CLASS_PAGE + 104)
 
 
-/* Must be a power of 2 for MPU alignment in ARMv7-M */
-#ifndef UVISOR_PAGE_SIZE
```

Comment:
How much work is needed still? Let us know when is the best moment to review the whole PR